src/webgpu/api/operation/vertex_state/correctness.spec.ts

```diff
@@ -941,13 +941,15 @@ g.test('max_buffers_and_attribs')
   .params(u => u.combine('format', kVertexFormats))
   .fn(t => {
     const { format } = t.params;
-    const attributesPerBuffer = Math.ceil(kMaxVertexAttributes / kMaxVertexBuffers);
+    // In compat mode, @builtin(vertex_index) and @builtin(instance_index) each take an attribute
+    const maxVertexAttributes = t.isCompatibility ? kMaxVertexAttributes - 2 : kMaxVertexAttributes;
```

Comment:
Is it possible to query the device for this info? If not this is fine for now, but a TODO to make like a separate limit tables or something for compat is probably a good idea!

Reply:
`device.limits.maxVertexAttribute` will be 16 on both compat and non-compat. The compat limit is that the 2 builtins each count as 1 attribute. So this test, knowing that it is using those 2 builtins, needs to subtract 2. If it wasn't using the builtins then it would not need to subtract them.